i915.c

```diff
@@ -1258,7 +1258,7 @@ static void *i915_bo_map(struct bo *bo, struct vma *vma, uint32_t map_flags)
 	    (bo->meta.format_modifier == I915_FORMAT_MOD_Y_TILED_GEN12_RC_CCS))
 		return MAP_FAILED;
 
-	if (i915->has_mmap_offset) {
+	if (i915->has_mmap_offset && bo->meta.tiling != I915_TILING_NONE) {
```

Comment:
why linear buffer can't use has_mmap_offset map?

Reply:
it can be used. just with slow performance in Buffer-Copy